src/app/auth/features/user-manager-detail/user-manager-detail.component.ts

```diff
@@ -33,37 +57,158 @@ export class UserManagerDetailComponent implements OnInit {
   showTransition = false;
 
   #route = inject(ActivatedRoute);
-  #router = inject(Router);
   #userService = inject(UserService);
 
+  #location = inject(Location);
+
   #userId = computed(() => this.#route.snapshot.params['id']);
```

Comment:
```suggestion
  id = input.required<string>();
```

And change #userId to id in the component. Then you don't need to inject Route.